starknet_py/tests/e2e/client_devnet/fixtures/accounts.py

```diff
@@ -0,0 +1,52 @@
+from typing import Tuple
+
+import pytest
+import pytest_asyncio
+
+from starknet_py.devnet.devnet_client import DevnetClient
+from starknet_py.net.account.account import Account
+from starknet_py.net.account.base_account import BaseAccount
+from starknet_py.net.models import StarknetChainId
+from starknet_py.net.signer.stark_curve_signer import KeyPair
+
+
+@pytest.fixture(scope="package")
+def account_devnet(
```

Comment:
Is there any specific reason not to use the existing `account` fixture?

Reply:
I wanted to have an account with a client of DevnetClient type, but this isn't used for now. I will delete this